docs/mp/integrations/wls.adoc

```diff
@@ -0,0 +1,392 @@
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+
+= Oracle WebLogic Server 
+:description: Helidon Oracle WebLogic Server Integration
+:keywords: wls, jms, rest, single sign-on, sso, microtx, web services
+:feature-name: Oracle WebLogic Server Integration Integration
+:rootdir: {docdir}/../..
+
+include::{rootdir}/includes/mp.adoc[]
+
+== Contents
+
+- <<Overview, Overview>>
+- <<REST Services Integration, REST Services Integration>>
+- <<JMS Integration, JMS Integration>>
+- <<Web Services Integration, Web Services Integration>>
+- <<Single Sign-On on OCI Integration, Single Sign-On on OCI Integration>>
+
+== Overview
+
+The Oracle WebLogic Server and Helidon integration enables interaction between a Helidon microservice application and an application installed on Oracle WebLogic Server, which is deployed in a Kubernetes cluster managed by the WebLogic Kubernetes Operator (WKO).
+
+The following integrations are available for your use:
+
+* The REST integration allows Oracle WebLogic Server applications and Helidon microservices to communicate through RESTful web service invocations, in one or both directions.
+* The JMS integration allows Oracle WebLogic Server applications and Helidon microservices to communicate through messaging, in one or both directions.
+* The Oracle WebLogic Web Services integration allows Helidon microservices to interact with Oracle WebLogic Server applications through SOAP (Simple Object Access Protocol) web service calls from Helidon to Oracle WebLogic Server.
+* The Oracle WebLogic Server and Helidon integration allows communication between a WebLogic cluster-hosted application and a Helidon microservice application by implementing Single Sign-on (SSO) authentication using Oracle Identity Cloud Service (IDCS).
+
+
+== REST Services Integration [[REST-Services]]
+
+- <<REST-Services-Overview, Overview>>
+- <<REST-Services-Maven-Coordinates, Maven Coordinates>>
+- <<REST-Services-Usage, Usage>>
+- <<REST-Services-References, References>>
+
+=== Overview [[REST-Services-Overview]]
+The REST services integration with Helidon enables bidirectional REST calls between Helidon and Oracle WebLogic Server. The Oracle WebLogic Server applications and Helidon microservices communicate through RESTful Web Service invocations, in one or both directions.
+
+=== Maven Coordinates [[REST-Services-Maven-Coordinates]]
```

Comment:
Does it make sense to point users to our CLI or Project Starter and, after that, explain the dependencies?